### ISSUE TITLE
Play off chain

### ISSUE THREAD

**Issue by Pepperrs** [ise-ethereum/on-chain-chess#70]
# Story

## User story

As a user, I want to be able to play chess without having to wait for each turn to be mined.

## Acceptance criteria

- [ ] The user can start a game of chess as they is used to
- [ ] The turns are not mined
- [ ] The contract can resolve turn issues
- [ ] The contract is informed of a winner
- [ ] Each user is able to inform the contract if the enemy has rage quit

**Comment by ljrahn** [comment]
Does using state channels for this make sense?

**Comment by graup** [comment]
> Does using state channels for this make sense?

Yeah, I think that could work. I haven't personally worked with it tho.
